fixtures/article.ts

```diff
@@ -972,14 +972,22 @@ export const data = {
                     },
                 ],
             },
+            hbImpl: 'prebid',
+            disableStickyTopBanner: false,
         },
         meta: {
-            isImmersive: false,
-            isHosted: false,
-            shouldHideAds: true,
-            hasStoryPackage: true,
             hasRelated: true,
+            hasShowcaseMainElement: false,
```

Comment:
Can we group all of the properties being added for commercial and add a comment? The reason this might be helpful is that we are likely to split the data model at some point (for AMP, but also questions about Editions although it seems like they're doing their own thing), so it's helpful to have clarity before then if that makes sense?

Reply:
OK I agree being able to identify what's used by commercial in the `model` at a glance would be great. `article.ts` is just a representation of the model sent from `frontend`, and I didn't create a specific grouping for the properties used by commercial in that model as some of those properties were already present elsewhere in the model, so if I were to create a comprehensive  commercial-specific grouping it'd mean duplicating them. I also imagine many of the properties added now to support commercial that would go into this commercial grouping will need to be used elsewhere at some point.

I could move all the new properties used by `graun.dotcom-rendering-commercial.js` into a commercial grouping, and duplicate the shared properties for now if we think it'll make splitting up the model in the near-future easier?